src/components/confusionMatrix/confusionMatrix.component.ts

```diff
@@ -8,7 +8,7 @@ class ConfusionMatrix {
     public body: any;
     public matrix: any;
     public keys: any;
-    public borderColor: string = '2px solid orange';
+    public borderColor: string = '2px solid #F9B42F';
```

Comment:
Couldn't this style be defined in a SCSS file?

Reply:
No, this variable is part of the highlights of rows and columns that are controlled inside the ts file.

Reply:
You can use `ng-class` to control

Reply:
It requires a big refactoring to be done, the proposal of this PR is to solve this bug #151370. So I haved opened this issue #45.